src/docs.md

```diff
@@ -9,11 +9,48 @@ description: The landing page for Flutter documentation.
         <div class="card-body">
             <header class="card-title">{{card.name}}</header>
             <p class="card-text">{{card.description}}</p>
-        </div>    
+        </div>
     </a>
 {% endfor %}
 </div>
 
+## What's new on flutter.io
+
+<b>Oct 31, 2018</b>
+
+Welcome to the revamped Flutter website!
+
+We've spent the last few months redesigning the website and how its
+information is organized. We hope you can more easily find the docs
+you are looking for. Some of the changes to the website include:
+
+* New [landing](/) page
+* New [showcase](/showcase) page
+* New navigation in the left side bar
+* New table of contents on the right side of most pages
+* New 404 page (_not yet_)
+* A [sitemap](/sitemap) (_not yet_)
+* New custom search  (_not yet_)
+
+Some of the new and updated content includes:
```

Comment:
In the list below, I'd drop the "New" and "A new" qualifiers. Again, I'd keep this list short.

If we feel the need to list all (most) changes, then maybe we need a webdev-like [Changelog](https://webdev.dartlang.org/changelog) page.